packages/typespec-test/test/keyVault.Administration/generated/typespec-ts/review/keyvault-admin.api.md

```diff
@@ -0,0 +1,415 @@
+## API Report File for "@azure/keyvault-admin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { ClientOptions } from '@azure-rest/core-client';
+import { OperationOptions } from '@azure-rest/core-client';
+import { OperationState } from '@azure/core-lro';
+import { PathUncheckedResponse } from '@azure-rest/core-client';
+import { Pipeline } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type ContinuablePage<TElement, TPage = TElement[]> = TPage & {
+    continuationToken?: string;
+};
+
+// @public
+export type DataAction = string;
+
+// @public
+export type ErrorModel = {
+    code?: string;
+    message?: string;
+    innerError?: ErrorModel_1;
+} | null;
+
+// @public
+export type ErrorModel_1 = {
```

Comment:
@maorleger this PR can fix the generation error, but it will have duplicate models issue because of this https://github.com/Azure/typespec-azure/issues/1677#issuecomment-2612003284 And @tadelesh will fix that after the holiday in tcgc. will you be okay with this fix for now and remove the duplicated models while you prepare the PR ? And we will fix this in emitter side after the holiday.
If you agree with this, I will try to release a new emitter with the current fix before the holiday.

Reply:
@qiaozha thanks for the heads up - this will be fine for me as KeyVault's generated code is not directly exported. We have a convenience layer and can smooth over any duplication as this. Thank you!